### PR TITLE
Add scenario to create a task attachment

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

File: src/main/java/core/api/ApiHeaders.java [hunk 1]
```diff
@@ -17,8 +17,10 @@
     ACCEPT("Accept"),
     APPLICATION_JSON("application/json"),
     CONTENT_TYPE("Content-Type"),
+    MULTIPART_FORM_DATA("multipart/form-data"),
     X_WWW_FORM_URLENCODED("application/x-www-form-urlencoded");
 
+
```
Comment (GustavoZHA) [inline src/main/java/core/api/ApiHeaders.java]: delete blank line

File: src/main/java/core/api/ApiRequest.java [hunk 2]
```diff
@@ -13,6 +13,8 @@
 import io.restassured.http.ContentType;
 import io.restassured.http.Header;
 import io.restassured.http.Headers;
+
```
Comment (GustavoZHA) [inline src/main/java/core/api/ApiRequest.java]: Delete extra line

File: src/main/java/core/api/ApiRequest.java [hunk 3]
```diff
@@ -289,4 +293,37 @@ public ContentType getContentType() {
...
+    /**
+     * Gets the multipart File
+     *
+     * @return the multipart file
+     * @author Jorge Caceres
+     */
+    public File getMultipartFile() {
+        return multipart.getFile();
+    }
+
```
Comment (GustavoZHA) [inline src/main/java/core/api/ApiRequest.java]: Delete blank line

File: src/test/java/cucumber/runner/RunTests.java [hunk 4]
```diff
@@ -48,11 +54,15 @@ public void setBaseEnv() throws JsonProcessingException {
         scenarioContext.setBaseEnvironment("list_in_space_id", listsRequest.createListInSpace());
         scenarioContext.setBaseEnvironment("task_id", tasksRequests.createTask());
         scenarioContext.setBaseEnvironment("checklist_id", checklistRequest.createChecklist());
+        fileOutputStream = new FileOutputStream("attachment.txt");
```
Comment (GustavoZHA) [inline src/test/java/cucumber/runner/RunTests.java:57]: "attachment.txt" is hard code?

File: src/test/resources/features/attachment/create_task_attachment.feature [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Feature: create a task attachment
+
+  @API @SmokeTesting @CreateTask @DeleteTask
+  Scenario: As a user I want to add dependency to a task
+    Given I set the endpoint /task/{task_id}/attachment
+    When I Add an attachment to the task
+    And I execute a POST request
```
Comment (GustavoZHA) [inline src/test/resources/features/attachment/create_task_attachment.feature]: add space

File: src/test/resources/features/attachment/create_task_attachment.feature [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Feature: create a task attachment
+
+  @API @SmokeTesting @CreateTask @DeleteTask
+  Scenario: As a user I want to add dependency to a task
```
Comment (GustavoZHA) [inline src/test/resources/features/attachment/create_task_attachment.feature]: change name of scenario

File: src/test/java/cucumber/runner/RunTests.java (hunk 4 above)
Comment (JorgeRCV) [reply src/test/java/cucumber/runner/RunTests.java:57]: Yeah for this cases is needed for the file added on the multipart requests

**Review by Raymundogs — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**